### ISSUE TITLE
Add SECURITY.md to templates

### ISSUE THREAD

**Issue by sed-i** [canonical/charmcraft#1125]
### What needs to get done

We should probably add a [SECURITY.md](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) file to all templates.

Or link to a [company-wide](https://github.com/canonical/.github) SECURITY.md ([example](https://github.com/microsoft/.github/blob/main/SECURITY.md)).

Cross-posted: https://github.com/canonical/.github/issues/6

### Why it needs to get done

This is becoming standard practice.

**Comment by lengau** [comment]
This is something we'll take care of team-wide via https://github.com/canonical/starbase/issues/82, which will get applied once we update charmcraft with those starbase changes.

**Comment by sed-i** [comment]
@sergiusens yeah we should rethink if we really want this as part of a charmcraft template.
